src/documents/management/commands/document_consumer.py

```diff
@@ -342,7 +342,7 @@ def handle_inotify(self, directory, recursive, is_testing: bool):
                 # If files are waiting, need to exit read() to check them
                 # Otherwise, go back to infinite sleep time, but only if not testing
                 if len(notified_files) > 0:
-                    timeout = inotify_debounce
+                    timeout = inotify_debounce * 1000
```

Comment:
might make sense to rename timeout to timeout_ms to make the code more explicit - just wanted to keep the change as small as possible.